api/controllers/Irc_EntryController.js

```diff
@@ -63,23 +63,35 @@ module.exports = {
   },
 
   process_event: function (request_body) {
-    if (request_body.operation === 'check in' || request_body.operation === 'update individual') {
-      return this.processEventDetaineeCreateOrUpdate(request_body);
+    if (request_body.operation === 'check in') {
+      return this.processEventDetaineesCreateOrUpdate(request_body);
+    }
+    if (request_body.operation === 'update individual') {
```

Comment:
you're only handling check in events so you can remove all mention of `update individual`